doc/source/whatsnew/v1.5.0.rst

```diff
@@ -814,6 +814,7 @@ I/O
 - Bug in :func:`read_excel` when reading a ``.ods`` file with newlines between xml elements (:issue:`45598`)
 - Bug in :func:`read_parquet` when ``engine="fastparquet"`` where the file was not closed on error (:issue:`46555`)
 - :meth:`to_html` now excludes the ``border`` attribute from ``<table>`` elements when ``border`` keyword is set to ``False``.
+- Most I/O methods do no longer suppress ``OSError`` and ``ValueError`` when closing file handles (:issue:`47136`)
```

Comment:
see https://github.com/pandas-dev/pandas/issues/47136#issuecomment-1140222578

Reply:
If it should go for 1.4, I can make a PR just for 1.4. Literally just removing the try-catch (1.4 doesn't have the problematic tar support; and the cleanup doesn't need to be part of 1.4 either).

Reply:
prob worth a targeted patch for 1.4

Reply:
opened #47160. Will move the whatsnew to 1.4.3 in this PR to keep the PRs in sync.